apps/patient/src/views/Main.tsx

```diff
@@ -55,6 +54,22 @@ export const Main = () => {
   const location = useLocation();
   const [faqModalIsOpen, setFaqModalIsOpen] = useState(false);
 
+  useEffect(() => {
+    if (location.pathname === '/' && token) {
```

Comment:
why do we only want to emit the event if the path is a slash? wouldn't we rather have this useEffect run on every page load (maybe add the url in as metadata lol?)

Reply:
i would just add a comment explaining why this is necessary, this seems like one of those innocuous things that might be easy to misunderstand/delete then subsequently polute our data